src/main.cpp

```diff
@@ -102,6 +122,30 @@ void setup()
         }
         delay(100);
     }
+    //set firingMode
+    if (pins.select1) {
+        select1.update();
+    }
+    if (pins.select2) {
+        select2.update();
+    }
+    if(select1.isPressed()){
+        firingMode = 0;
+    }
+    else if(select2.isPressed()){
+        firingMode = 2;
+    }
+    else {
+    firingMode = 1;
+    }
+    //changeFPS on boot by firing mode
+    for (int i = 0; i<numMotors; i++){
+        revRPM[i] = revRPMset[firingMode][i/2];
+        idleRPM[i] = idleRPMset[firingMode][i/2];
```

Comment:
This code relies on behavior that we have not defined earlier.

AKA this breaks if the motors are in the order of `first stage` then `second stage` (or vice versa).

If we move forward with this code change, we'll need to be explicit about this earlier

Reply:
True, the division was added to shorten the revRPMSet array from 3x4 to 3x2 in Configuration.h but since there are limitations for motor positions it's a UX decision to discuss with @ahalekelly which may not be worth it

Reply:
IMO, saving 18 bytes (6 extra bytes * 3 configurations) isn't a huge optimization, especially after we stepped up to the larger 8 MB chip.

That said, those 6 bytes could start to add up if we don't put a cap on the number of configurations. I think there's a larger discussion on what rpm we need to store (and some validation so the users are at least warned before they shoot themselves in the foot)

Reply:
Yeah a couple bytes of storage is negligible, I think having per-stage rpm is better from a UX perspective, but we need an intermediary table to set which motor numbers comprise which stages

Reply:
Gonna switch back to per-motor rpm and we can revisit this in the future